projects/core/src/product/connectors/reviews/product-reviews.connector.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { Review } from '../../../occ/occ-models/occ.models';
+import { ProductReviewsAdapter } from './product-reviews.adapter';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ProductReviewsConnector {
+  constructor(protected adapter: ProductReviewsAdapter) {}
+
+  getList(productCode: string, maxCount?: number): Observable<Review[]> {
```

Comment:
I would prefer to rename this to `get`, as it gets reviews for a single product. The return type makes it implicit.

Reply:
ok 👍